telemetry/telemetry.go

```diff
@@ -37,26 +37,18 @@ const (
 	Title = `Enable Telemetry`
 	// Help is a predefined text to display on the Telemetry
 	// screen for interactive installations
-	Help = `Allow the Clear Linux* OS to collect anonymous reports
-to improve system stability? These reports only relate to operating
-system details - no personally identifiable information is collected.
-
-See http://clearlinux.org/features/telemetry for more information.
+	Help = `Allow Clear Linux* OS to collect anonymized system data and usage
+statistics for continuous improvement?  These reports only relate to
+operating system details - no personally identifiable information
+is collected.
 `
 
-	// HelpMarkdown is a predefined text to display on the Telemetry
-	// screen for interactive installations
-	// This version supports markdown
-	HelpMarkdown = `Allow the Clear Linux* OS to collect anonymous reports
-to improve system stability? These reports only relate to operating
-system details - no personally identifiable information is collected.
-
-See <a href="http://clearlinux.org/features/telemetry">http://clearlinux.org/features/telemetry</a> for more information.
-`
+	// TelemetryAboutURL is the URL to reference for telemetry details
+	TelemetryAboutURL = `https://clearlinux.org/documentation/clear-linux/concepts/telemetry-about`
 
 	// RequestNotice is a common text string to be displayed when enabling
 	// telemetry by default on local networks
-	RequestNotice = "NOTICE: Enabling Telemetry preferred by default on internal networks"
+	RequestNotice = `NOTICE: Enabling Telemetry preferred by default on internal networks`
```

Comment:
The missing period at the end is preventing the translation of this line.

Reply:
> The missing period at the end is preventing the translation of this line.

Good catch! Fixed and resubmitted.